cookbooks/accounts/recipes/default.rb

```diff
@@ -107,3 +107,6 @@
   action :modify
   members administrators.sort
 end
+
+# Required account for mediawiki cookbook
+user "wiki"
```

Comment:
This is not how you add an account! Please look at how the accounts cookbook works to understand what you need to do... I would expect it will just happen though if you have the wiki role loaded and run the `accounts::default` recipe.

Reply:
I’ll update, thanks. It didn’t just work, unfortunately. The ordering of the role/recipe run list is significant and I’ll reverse them and see if it helps and makes this line unnecessary.

Reply:
Following up, this really didn’t help and [immediately brought up an elasticsearch error](https://gist.github.com/migurski/fad477dbf80bc78bef1bfeba80397bdb):

         run_list:
           - role[wiki]
           - recipe[wiki]